python/ray/_raylet.pyx

```diff
@@ -576,13 +576,24 @@ cdef execute_task(
 
             backtrace = ray._private.utils.format_error_message(
                 traceback.format_exc(), task_exception=task_exception)
+
+            # Generate the actor repr from the handle if the failure was
+            # an actor creation task or actor task.
+            actor_handle = None
+            if <int>task_type != <int>TASK_TYPE_NORMAL_TASK:
+                actor_id = core_worker.get_actor_id()
+                actor_handle = core_worker.get_actor_handle(actor_id)
```

Comment:
I don't think you want the repr of the actor handle, but the repr of the `actual actor` (self).

Reply:
actual actor means the class itself, am I correct?

Reply:
Or you are talking about `ActorClass`?

Reply:
It looks like in this code, it is a bit tricky to get ActorClass, but we can easily get the original class. But in that case, repr is like this by default; `test_traceback.A object at 0x7fa0653b8750>`

Reply:
So, it looks like we can only obtain the original class here, but if so, the default looks like this`test_traceback.A object at 0x7fb517805710>`. But if users override their repr, that will be reflected. Does this look okay to you?

Reply:
Actually, this is the mechanism I am trying. Please lmk if you have other good ideas;

If the actor class repr is default => repr of actor handle
otherwise => actor class' repr

Reply:
The class itself, `self`, which is the only one containing useful information.

> istest_traceback.A object at 0x7fb517805710>. But if users override their repr, that will be reflected. Does this look okay to you?

It's fine. You could also report `repr=None` but that might be confusing.

Reply:
Here's the current implementation; 

If actor class doesn't specify repr (in this case, it will be like `test_traceback.A object at 0x7fb517805710>`); We capture it and format it in this way;
```
ray::A.f() (pid=70926, ip=192.168.1.5, repr=Actor(repr=A))
```

If actor class does (e.g., repr returns "ABC");
```
ray::A.f() (pid=70932, ip=192.168.1.5, repr=ABC)
```

Reply:
We can capture this by checking repr(actor) == object.__repr__(actor)

Reply:
This is weird. Can you either set it to None or use repr verbatim?

Reply:
Sure. I will then just print the original repr in this case